### PR TITLE
Cocoapods config for Artifactory repositories

### PR DESCRIPTION
**Pull request by barv-jfrog**
- [ ] All [tests](https://github.com/jfrog/jfrog-cli-core#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [ ] All [static analysis checks](https://github.com/jfrog/jfrog-cli-core/actions/workflows/analysis.yml) passed.
- [ ] This pull request is on the dev branch.
- [ ] I used gofmt for formatting the code before submitting the pull request.
-----
Description:
As part of support of cocoapods in jf audit, we want to enable configuration of an artifactory cocoapods repository that serves as the base of dependency installation in jf audit.

### CODE REVIEW

File: artifactory/commands/cocoapods/podcommand.go [hunk 1]
```diff
@@ -0,0 +1,177 @@
+package cocoapods
+
```
Comment (eyalbe4) [inline artifactory/commands/cocoapods/podcommand.go:2]: This command should be moved to be inside the [`jfrog-cli-security`](https://github.com/jfrog/jfrog-cli-security) go module.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: artifactory/commands/cocoapods/podcommand.go (hunk 1 above)
Comment (srmish-jfrog) [reply artifactory/commands/cocoapods/podcommand.go:2]: Why? This isn't the "audit" command, I didn't see any `SetArtifactoryAsResolutionServer` functions being defined in `jfrog-cli-security`, only here (`jfrog-cli-core`)

Comment (srmish-jfrog) [reply artifactory/commands/cocoapods/podcommand.go:2]: Will move it to reduce cross-team dependencies

**Comment by eyalbe4** [comment]
Closing this one, since it's been replaced by https://github.com/jfrog/jfrog-cli-core/pull/1288.

**Pull request closed without merging**